analyzer/src/main/kotlin/managers/Conan.kt

```diff
@@ -272,8 +275,12 @@ class Conan(
     /**
      * Return the value `conan inspect` reports for the given [field].
      */
-    private fun runInspectRawField(pkgName: String, workingDir: File, field: String): String =
-        run(workingDir, "inspect", pkgName, "--raw", field).stdout
+    private fun inspectField(pkgName: String, workingDir: File, field: String): String =
+        pkgInspectResults.getOrPut(pkgName) {
+            val jsonFile = createOrtTempFile(managerName)
+            run(workingDir, "inspect", pkgName, "--json", jsonFile.absolutePath).requireSuccess()
+            jsonMapper.readTree(jsonFile).also { jsonFile.delete() }
```

Comment:
Nit: Should we do `jsonFile.parentFile.safeDeleteRecursively(force = true)` instead, to delete the whole temporary directory?